### ISSUE TITLE
Build error from `toStdString` method.

### ISSUE THREAD

**Issue by rnburn** [flier/zipkin-cpp#3]
I get this error when building zipkin-cpp:
```
[  8%] Built target GTest
[ 16%] Built target DoubleConversion
[ 25%] Built target LibRDKafka
[ 33%] Built target RapidJSON
[ 41%] Built target GBench
[ 50%] Built target GFlags
[ 51%] Building CXX object src/CMakeFiles/zipkin.dir/KafkaCollector.cpp.o
/Users/rnickb/src/zipkin-cpp/src/KafkaCollector.cpp:96:38: error: no member named 'toStdString' in
      'std::__1::basic_string<char>'
        initial_brokers = uri.host().toStdString();
                          ~~~~~~~~~~ ^
/Users/rnickb/src/zipkin-cpp/src/KafkaCollector.cpp:108:70: error: no member named 'toStdString' in
      'std::__1::basic_string<char>'
            compression_codec = parse_compression_codec(param.second.toStdString());
                                                        ~~~~~~~~~~~~ ^
/Users/rnickb/src/zipkin-cpp/src/KafkaCollector.cpp:112:62: error: no member named 'toStdString' in
      'std::__1::basic_string<char>'
            message_codec = MessageCodec::parse(param.second.toStdString());
                                                ~~~~~~~~~~~~ ^
3 errors generated.
```

 I would think you'd want to use the `toStdString` [function](https://github.com/facebook/folly/blob/master/folly/String.h#L44) instead of the method for conversions which will work regardless of whether it's called with an std or folly string.

**Comment by flier** [comment]
Thanks, fixed in [1690d8a](https://github.com/flier/zipkin-cpp/commit/1690d8ab052591a9789494878b9299b164a14bd6), will be release later after I fix the broken traivs CI build.

**Comment by flier** [comment]
merged into [v0.3.1](https://github.com/flier/zipkin-cpp/releases/tag/v0.3.1).
